GliaWidgets/Sources/Extensions/UIImage+Extensions.swift

```diff
@@ -54,4 +54,24 @@ extension UIImage {
             orientation: self.imageOrientation
         )
     }
+
+    func image(withTintColor color: UIColor) -> UIImage {
```

Comment:
Is there a reason why this is needed instead of setting the tintColor of the imageView ?

Reply:
Yes, the reason is that `UIButton.setImage(_ image: UIImage?, for state: UIControl.State)` requires `UIImage` instance. Since image colour is specified via styling for different button states, multiple images with different colors should be provided.